src/main/java/sirius/db/text/ReduceCharacterProcessor.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.db.text;
+
+import sirius.kernel.commons.Strings;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * Reduces characters to a simpler representation.
+ * <p>
+ * Many characters like umlauts or ligatures are "down converted" by this processor into their ASCII representation.
+ */
+public class ReduceCharacterProcessor extends ChainableTokenProcessor {
```

Comment:
```suggestion
public class UnicodeToAsciiProcessor extends ChainableTokenProcessor {
```

Reply:
unicode is kind of a big word here - we transform something like 40 of 65k characters :-P